src/API/HealthController.ts

```diff
@@ -39,13 +66,49 @@ export class HealthController {
     }
   }
 
+  private async getBlockchainInfo(): Promise<any> {
+    try {
+      const { blockchainInfo } = (await this.collection.findOne({ name: 'blockchainInfo' })) || {
+        blockchainInfo: 'Getting Blockchain Info...',
+      }
+      return blockchainInfo
+    } catch (e) {
+      return e
```

Comment:
I am not sure we will want to expose the error through the api like this. @lautarodragan thoughts on what you would like to have returned?

Reply:
I changed these to return 'Error retrieving `x`'